packages/devtools_app/lib/src/html_message_manager.dart

```diff
@@ -46,8 +46,7 @@ class HtmlMessageManager {
       _messages[screenId]?.remove(_message);
     });
 
-    // ignore: prefer_collection_literals
-    _messages.putIfAbsent(screenId, () => Set()).add(message);
+    _messages.putIfAbsent(screenId, () => {message});
```

Comment:
This changed behavior. Was that intentional?

Reply:
good catch! that is a bug.